.rr.yaml

```diff
@@ -36,4 +36,74 @@ http:
   http2:
     enabled: false
     h2c: false
-    maxConcurrentStreams: 128
\ No newline at end of file
+    maxConcurrentStreams: 128
+
+redis:
+  # UniversalClient is an abstract client which - based on the provided options -
+  # can connect to either clusters, or sentinel-backed failover instances
+  # or simple single-instance servers. This can be useful for testing
+  # cluster-specific applications locally.
+  # if the number of addrs is 1 and master_name is empty, a single-node redis Client will be returned
+
+  # if the number of Addrs is two or more, a ClusterClient will be returned
+  addrs:
+    - 'localhost:6379'
+  # if a MasterName is passed a sentinel-backed FailoverClient will be returned
+  master_name: ''
+  username: ''
+  password: ''
+  db: 0
+  sentinel_password: ''
+  route_by_latency: false
+  route_randomly: false
```

Comment:
Please use consistent parameter naming. See http config above.

Reply:
Yep, agree, I'll move all config keys from snake to underscore notation. Issue https://github.com/spiral/roadrunner/issues/458